vignettes/estimate_time_varying_severity.Rmd

```diff
@@ -312,23 +279,23 @@ df_covid_cfr <- mutate(
 df_covid_cfr <- unnest(df_covid_cfr, cols = data)
 ```
 
-For simplicity, we use the same delay distribution between onset and death for all locations.
+For simplicity, we use the same delay distribution between onset and death for all countries.
 
 Finally we plot the time-varying CFR for a selection of three more countries with large outbreaks of COVID-19: Brazil, India, and the United States.
 
 ```{r, fig.cap = "Example plot of the corrected time-varying CFR. We calculate the time-varying CFR for the ongoing COVID-19 epidemic in Brazil, India, and the United States, corrected for delays.", class.source = 'fold-hide', fig.width=8}
-filter(df_covid_cfr, location %in% c("Brazil", "India", "United States")) %>%
+filter(df_covid_cfr, country %in% c("Brazil", "India", "United States")) %>%
```

Comment:
I get an error here:
```
Error in `filter()`:
! Problem while computing `..1 = country %in%
  c("Brazil", "India", "United States")`.
Caused by error in `country %in% c("Brazil", "India", "United States")`:
! object 'country' not found
```
This is what I have for column names for that object:
```
> names(df_covid_cfr)
[1] "date"   "cases"  "deaths" ".by"   
```

Reply:
Thanks. This is now working – looks like old version of {cli} was obstructing the {tidyr} update (a quick search this has cropped up for others with RStudio too).